panel/io/server.py

```diff
@@ -210,6 +212,16 @@ def get_server(panel, port=0, websocket_origin=None, loop=None,
     if isinstance(panel, dict):
         apps = {}
         for slug, app in panel.items():
+            if isinstance(title, dict):
+                try:
+                    title_ = title[slug]
+                except KeyError:
+                    raise KeyError(
+                        "Keys of the title dictionnary and of the apps "
+                        f"dictionnary must match. No {slug} key found in the "
```

Comment:
```suggestion
                        f"dictionary must match. No {slug} key found in the "
```

At some point we should convert to f-strings throughout.